frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboardView-filters.stories.tsx

```diff
@@ -464,6 +465,7 @@ LightThemeTextWithValue.play = async ({ canvasElement }) => {
     "filter value",
   );
   await userEvent.click(getLastPopoverElement());
+  asyncCallback();
```

Comment:
This was done to prevent one (2 if you count the dark theme variant) flaky loki test.

Since there's no way to stress test this. I just [reran the test 2 more times for a total of 3 times.](https://github.com/metabase/metabase/actions/runs/10212204940/job/28256805062?pr=46382). And they all passed 😉